cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeConfigurationBuilder.java

```diff
@@ -0,0 +1,342 @@
+     *
+     * @param bridgeId  the bridge ID
+     */
+    private void configureBridgeId(String bridgeId)   {
+        printSectionHeader("Bridge ID");
+        writer.println("bridge.id=" + bridgeId);
+        writer.println();
+    }
+
+    /**
+     * Renders the Apache Kafka bootstrap servers configuration
+     *
+     * @param bootstrapServers  Kafka cluster bootstrap servers to connect to
+     */
+    private void configureBootstrapServers(String bootstrapServers) {
+        printSectionHeader("Kafka bootstrap servers");
+        writer.println("kafka.bootstrap.servers=" + bootstrapServers);
+        writer.println();
+    }
+
+    /**
+     * Configure the Kafka security protocol to be used
+     * This internal method is used when the configuration is build, because the security protocol depends on
+     * TLS and SASL authentication configurations and if they are set
+     */
+    private void configureSecurityProtocol() {
+        printSectionHeader("Kafka Security protocol");
+        writer.println("kafka.security.protocol=" + securityProtocol);
+    }
+
+    /**
+     * Configures the Kafka config providers used for loading some parameters from env vars and files
+     * (i.e. user and password for authentication)
+     *
+     * @return  the builder instance
+     */
+    public KafkaBridgeConfigurationBuilder withConfigProviders() {
+        printSectionHeader("Config providers");
+        writer.println("kafka.config.providers=strimzienv,strimzifile,strimzidir");
+        writer.println("kafka.config.providers.strimzienv.class=org.apache.kafka.common.config.provider.EnvVarConfigProvider");
+        writer.println("kafka.config.providers.strimzienv.param.allowlist.pattern=.*");
+        writer.println("kafka.config.providers.strimzifile.class=org.apache.kafka.common.config.provider.FileConfigProvider");
+        writer.println("kafka.config.providers.strimzifile.param.allowed.paths=/opt/strimzi");
+        writer.println("kafka.config.providers.strimzidir.class=org.apache.kafka.common.config.provider.DirectoryConfigProvider");
+        writer.println("kafka.config.providers.strimzidir.param.allowed.paths=/opt/strimzi");
+        writer.println();
+        return this;
+    }
+
+    /**
+     * Adds the tracing type
+     *
+     * @param tracing   the tracing configuration
+     * @return  the builder instance
+     */
+    public KafkaBridgeConfigurationBuilder withTracing(Tracing tracing) {
+        if (tracing != null) {
+            printSectionHeader("Tracing configuration");
+            writer.println("bridge.tracing=" + tracing.getType());
+            writer.println();
+        }
+        return this;
+    }
+
+    /**
+     * Adds the TLS/SSL configuration for the bridge to connect to the Kafka cluster.
+     * The configuration includes the trusted certificates store for TLS connection (server authentication)
+     * as well as the keystore configuration for mTLS (client authentication).
```

Comment:
this method seems to only configure truststore, but not the keystore as the doc suggests

Reply:
Good catch! It's a leftover when the method did that, but not anymore because the logic is all within the `withAuthentication` method.